pkg/util/log/slogadapter.go

```diff
@@ -12,20 +12,20 @@ import (
 // SlogFromGoKit returns slog adapter for logger.
 func SlogFromGoKit(logger log.Logger) *slog.Logger {
 	var sl slog.Level
-	x, ok := logger.(leveledLogger)
-	if !ok {
+	x := privateLevelDetector{
+		string:   "This struct is expected to be used with levelLogger only",
+		logLevel: debugLevel,
+	}
+	logger.Log("test", &x)
```

Comment:
Could you add a one-liner code comment with what this "test" does? From the initial pass over the code, I thought it was a left-over, and it took me a while to process it.

Reply:
Done; also changed the string to "probe".  Let me know what you think.